app/src/main/res/values/strings.xml

```diff
@@ -42,6 +42,10 @@
     <string name="settings_backup_storage_code_dialog_message">To enable storage backup, you need to first verify your recovery code or generate a new one.</string>
     <string name="settings_backup_storage_code_dialog_ok">Verify code</string>
 
+    <string name="settings_expert_title">Expert settings</string>
+    <string name="settings_expert_quota_title">Unlimited app quota</string>
+    <string name="settings_expert_quota_summary">Does not impose a limitation on the size of app backups.\n\nWarning: This can fill up your storage location quickly and should not be needed. Apps that exceed their quota have a bug that should get fixed.</string>
```

Comment:
Made some minor changes:

Do not impose a limitation of the size of app backups.\n\nWarning: This call fill up your storage location quickly. It should not be needed in most cases, and apps that exceed their quota usually have a bug that should get fixed.

Reply:
`impose a limitation on the size` sounds more correct to me, but I am not a native speaker

Using the `, and apps that ` makes the sentence longer. Usually better for readability to have more shorter sentences than few long ones.

Reply:
> `impose a limitation on the size` sounds more correct to me, but I am not a native speaker
> 
Ah yes I did mean 'on', that was a 2AM typo. The main change there was Does -> Do

> Using the `, and apps that ` makes the sentence longer. Usually better for readability to have more shorter sentences than few long ones.

Fair.

"Do not impose a limitation on the size of app backups.\n\nWarning: This can fill up your storage location quickly. Not needed for most apps."

Left out the part about bugs.